packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -516,6 +757,12 @@ abstract class SliderComponentShape {
 
   /// Paints the shape, taking into account the state passed to it.
   ///
+  /// [context] is the context of the render box of the [Slider]. The origin of
```

Comment:
The [center] offset is relative to the origin of the canvas.

Reply:
I think this comment may have to be adjusted with the macros that are now being used